src/utils/layout-helpers.ts

```diff
@@ -117,10 +120,26 @@ export function getPodModulePrefix(root: string): string | null {
   return podModulePrefix.trim().length > 0 ? podModulePrefix : null;
 }
 
+export function getRoot(fromDir: string) {
+  try {
+    const topLevel = childProcess.execSync('git rev-parse --show-toplevel', { encoding: 'utf8', cwd: fromDir });
+
+    return topLevel.trim();
+  } catch {
+    // we ignore the error, because we might not be in a git repo
+    return '/';
+  }
+}
+
 export async function resolvePackageRoot(root: string, addonName: string, packagesFolder = 'node_modules'): Promise<string | false> {
   const roots = root.split(path.sep);
+  const stopAt = getRoot(root);
```

Comment:
it's not correct for case, where we have monorepos, with shared packages on top.

like:
```
node_modules
package.json
frontend-apps
__my-app
```

Reply:
Isn't that the git root tho?
🤔

Reply:
@NullVoxPopuli we may not have it browser env (if we browse code from github), also it seems project may have few git folders is submodules used https://git-scm.com/docs/gitsubmodules, in addition - it's +1 fs call (to check for `.git` foder)

Reply:
that makes sense -- thanks!